Finland_adb.txt

```diff
@@ -873,6 +873,23 @@ murobbs.muropaketti.com###hintaopas_top_product_block
 ##[src*="://ad.doubleclick.net/"]
 ||cloudfront.net/banners/$domain=tori.fi
 
+! Kotikokki.net ad containers
+kotikokki.net##.partners
+
+! Ylilauta.org ad containers
+ylilauta.org##:xpath(//div[@id="adform-outstream"]/parent::*)
+
+! utag
+*/utag/sanoma-fi/$script
```

Comment:
Examples of URLs this is used on?

Reply:
Most sanoma sites list here:
https://sanoma.fi/tietoa-meista/tuotteet/